docs/content/en/docs/installation/configuration/namespace.md

```diff
@@ -31,7 +31,7 @@ Some considerations for Keptn:
   resources that are used for pre- and post-deployment
   can reference metrics from any namespace.
   So you can create `KeptnMetrics` in a centralized namespace
-  (such as `keptn-lifecycle-toolkit`)
+  (such as `keptn-lifecycle-toolkit-system`)
```

Comment:
```suggestion
  (such as `keptn-system`)
```

Reply:
This will be fixed in a follow-up https://github.com/keptn/lifecycle-toolkit/pull/2565